set-static-ip

```diff
@@ -29,7 +29,7 @@ if [ -n "$PROVISIONING_INTERFACE" ]; then
   fi
 
   # Get rid of any DHCP addresses on the dedicated provisioning interface
-  /usr/sbin/ip address flush dev "$PROVISIONING_INTERFACE"
+  /usr/sbin/ip address flush dev "$PROVISIONING_INTERFACE" scope global
```

Comment:
can we include a comment explaining why we are only flushing global scope addresses? or a pointer back to jira bug is sufficient too.

Reply:
@abhat the pointer to the jira bug is already present in the PR title
I will add a commit message as suggested by Zane though, forgot about it